src/test/java/Homework20.java

```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+public class Homework20 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        String expectedPlaylistDeletedMessage = "Deleted playlist \"Homework17.\"";
+
+        navigateToPage();
+        provideEmail("john.wolschleger@testpro.io");
+        providePassword("te$t$tudentb49");
+        clickSubmit();
+        //Thread.sleep(2000);
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.